.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       SKIP_DOCKER: true
     steps:
       - base-install
-      - run: ./packages/<< parameters.package >>/scripts/test-ci.sh
+      - run: ./.circleci/test-package.sh << parameters.package >>
```

Comment:
I snuck this in. Going directly to `test-ci.sh` skips the check for whether this package needs to be tested. This change fixes that.